### PR TITLE
Web frontend gps bugfix

### PR DESCRIPTION
**Pull request by dexterastin**
Troubleshooting for an unaccessible GPS access. 😛

### CODE REVIEW

File: static/js/initCardMapView.js [hunk 1]
```diff
@@ -2,81 +2,13 @@ var places = [];
...
-
-                        drop();
-                    }
-                }
-            );
-
-
-        })
+String.prototype.format = function () {
+    a = this;
```
Comment (shberry) [inline static/js/initCardMapView.js]: can you make the variable name more meaningful?

File: static/js/initCardMapView.js [hunk 2]
```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
         markers[i].setMap(null);
     }
     markers = [];
-}
\ No newline at end of file
+}
+
```
Comment (shberry) [inline static/js/initCardMapView.js:57]: remove extra line

File: static/js/initCardMapView.js [hunk 3]
```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
...
+
+if (navigator.geolocation) {
```
Comment (shberry) [inline static/js/initCardMapView.js]: is this if/else part of production code?

File: static/js/initCardMapView.js [hunk 4]
```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
...
+        urlAPI = "http://54.241.216.252:5000/mappedcafes/{0}/{1}/{2}"
+            .format(
+                token,
+                startPos.coords.latitude,
+                startPos.coords.longitude
+            );
+
+        tmpUrl = "http://54.241.216.252:5002/mappedcafes/4235c90663f34d6bb90d4e2c8e2bf875";
+
+        console.log(urlAPI);
```
Comment (shberry) [inline static/js/initCardMapView.js]: is this part of production code?

File: static/js/initCardMapView.js [hunk 5]
```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
...
+                    var activeCard = null;
+
+                    setInterval(function (e) {
+                        var currentActiveCard = $('.active');
+                        if (activeCard !== currentActiveCard) {
+                            activeCard = currentActiveCard;
+                            activeCard.trigger("click");
+                        }
+                    }, 500);
+
```
Comment (shberry) [inline static/js/initCardMapView.js]: i think it's okay to remove this extra line

File: static/js/initCardMapView.js [hunk 6]
```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
...
+                    drop();
+                }
+            }
+        );
+
```
Comment (shberry) [inline static/js/initCardMapView.js]: remove extra line

**Review by shberry — COMMENTED** [review]
Left a few comments. Thanks!

File: static/js/initCardMapView.js (hunk 4 above)
Comment (dexterastin) [reply static/js/initCardMapView.js]: Yes. tmpUrl is test url. When API development is completed, I intend to change it.

File: static/js/initCardMapView.js (hunk 3 above)
Comment (dexterastin) [reply static/js/initCardMapView.js]: What's mean?

File: static/js/initCardMapView.js (hunk 1 above)
Comment (dexterastin) [reply static/js/initCardMapView.js]: ok! thank you!

**Review by khc196 — APPROVED** [review]

**Pull request merged**